app/views/development_metrics/projects.erb

```diff
@@ -32,7 +35,10 @@
     </div>
 
     <div class="metrics shadow-box">
-      <h4 class="p-3">Time to merge</h4>
+      <div class='row'>
+        <h4 class="p-3" style='margin-left:10px;'><%= @merge_time_definition && @merge_time_definition[:name] %></h4>
```

Comment:
you can do only `@merge_time_definition[:name]` because if the value don't exist it will return nil

Reply:
also, don't you need to pass the name of the metric in `:name`? how do you know what metric will show up there?